doc/sphinx-guides/source/developers/containers.rst

```diff
@@ -2,17 +2,38 @@
 Docker, Kubernetes, and Containers
 ==================================
 
-The Dataverse Community is exploring the use of Docker, Kubernetes, and other container-related technologies. The primary community-lead projects to watch are:
+The Dataverse community is exploring the use of Docker, Kubernetes, and other container-related technologies.
+
+.. contents:: |toctitle|
+	:local:
+
+Container Guide
+---------------
+
+We recommend starting with the :doc:`/container/index`. The core Dataverse development team, with lots of help from the community, is iterating on containerizing the Dataverse software and its dependencies there.
```

Comment:
Shouldn't we give a shout out to Oliver and the FZJ (without whom containers wouldn't be available)

Reply:
Technically I am part of the core team, so no extra shoutout necessary.

Reply:
I absolutely wrote this with the context that @poikilotherm is now a member of the core development team! 🎉 

The announcement was here: https://groups.google.com/g/dataverse-community/c/uDeNUfqe3RE/m/dph8fZUaEAAJ